### PR TITLE
feat(Catalog Management): get regions routes, support install_type param

### PR DESCRIPTION
**Pull request by benbuchanan**
## PR summary

Regenerating after some small API changes.  Need to support install_type param on reloadOffering function for a user.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Current vs new behavior  

Only adding new params and routes.

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

## Test results
Integration tests
```
Test Suites: 20 skipped, 1 passed, 1 of 21 total
Tests:       406 skipped, 51 passed, 457 total
Snapshots:   0 total
Time:        31.572 s
Ran all test suites matching /test\/integration\//i.
```

Example tests
```
Test Suites: 1 passed, 1 total
Tests:       47 passed, 47 total
Snapshots:   0 total
Time:        27.183 s
Ran all test suites matching /examples\/catalog-management.v1.test.js/i.
```

### CODE REVIEW

**Comment by benbuchanan** [comment]
Updated tests

Integration
```
Test Suites: 20 skipped, 1 passed, 1 of 21 total
Tests:       406 skipped, 52 passed, 458 total
Snapshots:   0 total
Time:        33.32 s
Ran all test suites matching /test\/integration\//i.
```

Examples
```
Test Suites: 1 passed, 1 total
Tests:       48 passed, 48 total
Snapshots:   0 total
Time:        27.688 s
Ran all test suites matching /examples\/catalog-management.v1.test.js/i.
```

File: catalog-management/v1.ts [hunk 1]
```diff
@@ -4192,6 +4213,67 @@ class CatalogManagementV1 extends BaseService {
...
+   * Validates deployment input variables.
+   *
+   * @param {Object} params - The parameters to send to the service.
+   * @param {string} params.versionLocId - A dotted value of `catalogID`.`versionID`.
+   * @param {string} [params.input1] - A deployment variable to validate.
+   * @param {string} [params.input2] - Another deployment variable to validate.
+   * @param {OutgoingHttpHeaders} [params.headers] - Custom request headers
+   * @returns {Promise<CatalogManagementV1.Response<CatalogManagementV1.VersionInputValidationResponse>>}
+   */
+  public validatesInputs(
```
Comment (padamstx) [inline catalog-management/v1.ts:4228]: This seems like an odd name for an operation.  Perhaps `validateInputs` would be a slightly better name?

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**